bgpd/bgp_labelpool.h

```diff
@@ -37,6 +37,7 @@ struct labelpool {
 	struct lp_fifo		*requests;	/* blocked on zebra */
 	struct work_queue	*callback_q;
 	uint32_t		pending_count;	/* requested from zebra */
+	int			lm_init;	/* label manager init. */
```

Comment:
Why do we need a new variable added to the labelpool data structure?  What's the long term usage?  In this code we only ever assign it from lm_label_manager_connect results.